### DISCUSSION TITLE
Progress bar changes size during post trial gap

### DISCUSSION THREAD

**Discussion by pandabar** [jspsych/jsPsych#2208] [Q&A]
Hi all, 
I have an experiment ready to go and everything seems right, except that the progress bar changes its size during the post trial gap, and I don't know why. The experiment is a procedure with two trials: the first one presents the audio stimulus with the pictures as options and the second one shows only the pictures (I did it like this so I could record the reaction times from when the stimulus stops playing). Here is the code:

```js
var stim2 = {
      type: 'audio-keyboard-response',
      stimulus: jsPsych.timelineVariable('stimulus2'),
      prompt: function(){
                var html="<img src='"+jsPsych.timelineVariable('pic', true)+"'>";
                html += "<p>Welches Wort haben Sie geh&ouml;rt?</p><div style='height:20px; width:100%'></div><div><img src='pics/buttons2.png'></div>";
                return html;
            },   
      choices: jsPsych.NO_KEYS,
      trial_ends_after_audio: true
}
var resp2 = {
    type: 'image-keyboard-response',
    choices: ['p', 'q' ],
    prompt: '<p>Welches Wort haben Sie geh&ouml;rt?</p>'+
  '<div style="height:20px; width:100%"></div>' + 
  '<div><img src="pics/buttons2.png"> </div>',
    trial_duration: 5000,
    post_trial_gap: 800,
    stimulus: jsPsych.timelineVariable('pic'),

    on_finish: function() {
        var curr_progress_bar_value = jsPsych.getProgressBarCompleted();
        jsPsych.setProgressBar(curr_progress_bar_value + (1/n_screens));
    }
}
var ident_proc = {
      timeline: [stim2, resp2],
      timeline_variables: test_stimuli2,
     sample: {
        type: 'fixed-repetitions',
        size: 3 // 3 repetitions of each trial, 12 total trials, order is randomized.
    }
 }

timeline.push(ident_proc);

 jsPsych.init({
	 timeline: timeline,
   preload_images: images,
   preload_audio: audio,
   show_progress_bar: true,
 message_progress_bar:"Fortschritt:",
 auto_update_progress_bar: false
 })
```

Thank you for your help!!

**Comment by jodeleeuw** [comment]
Hey @pandabar,

What part of the progress bar is changing size? Any chance you have a live version online that we could look at?

**Comment by pandabar** [comment]
Hi Josh,

Thanks for the quick reply! It's the entire size of the bar (as in, the aspect ratio); the progress itself is ok. However, the first task (discrimination) is fine; it's only the second one (identification) that looks funny. I commented out most of the experiment and left only two trials for each task: https://ling.sprachwiss.uni-konstanz.de/features/myexperiment_test.html

Thanks, 
Fernanda

**Reply by jodeleeuw** [reply]
Thanks for the link, and for a short version 😁 I didn't notice any changes that looked funny when I ran it. I'm testing with Google Chrome on Windows. What browser are you using?

**Reply by pandabar** [reply]
I'm using Firefox and Chromium on Linux. It's not really important though, it just makes it look like a "hiccup" and I thought it could be uncomfortable for the participants, in the event that they are actually paying attention to the bar... good to hear it's not noticeable on Chrome!
